spec/features/apply_to_du_spec.rb

```diff
@@ -14,6 +14,7 @@
 
     fill_in "Twitter username", with: "@beepboopbeep"
     fill_in "Blog URL", with: "http://blog.awesome.com"
+    fill_in "Pronouns", with: "They/Them"
```

Comment:
It would be good here to have a spec that proves that the pronouns that we filled in actually showed up on the form when saved.